playground/mocks/data/idp/idx/authenticator-verification-email-no-profile.json

```diff
@@ -52,7 +52,7 @@
             "type":"object",
             "options":[
               {
-                "label":"Primary Email Address",
+                "label":"Email",
```

Comment:
What about a test where the label is like "Custom Email" to check that "Email" is not being hardcoded.